OmiseSwift/API Models/Source.swift

```diff
@@ -4,6 +4,7 @@ import Foundation
 public enum Flow: String, Codable, Equatable {
     case redirect
     case offline
+    case app_redirect
```

Comment:
Can you rename it to:
```
case appRedirect = "app_redirect"
```
On iOS it's not very common to use an underscore.

Reply:
yes, will do